### ISSUE TITLE
[Request] Convert px to rem

### ISSUE THREAD

**Issue by coudrak** [leongersen/noUiSlider#1181]
I think it would be good to update the less files to replace all the css px with rem values.
Most of modern projects switched to rem, like Boostrap.

**Comment by leongersen** [comment]
Thanks for your suggestion. What advantage would you see in making this migration? 

Related to #1167.

**Comment by coudrak** [comment]
For example, in my case, I "zoom out" my page by 75% on mobile screens less than 320px wide, in this way:

```
:root {
   font-size: 75%;
}
```

I use Bootstrap, so everything in my page decreases, except the slider because it is in px.

**Comment by leongersen** [comment]
Right, that makes sense. Given that is would be a tricky breaking change, in particular for restyled sliders, it would be good to pick this up together with the issue linked above.
